<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>RegisterDialog</class>
 <widget class="QDialog" name="RegisterDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>460</width>
    <height>299</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <widget class="QLabel" name="id_label">
       <property name="text">
        <string>Your ID:</string>
       </property>
       <property name="buddy">
        <cstring>id_line</cstring>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="id_line">
       <property name="readOnly">
        <bool>true</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_7">
     <item>
      <widget class="QLabel" name="inviter_label">
       <property name="text">
        <string>Inviter ID:</string>
       </property>
       <property name="buddy">
        <cstring>secret_line</cstring>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="inviter_line">
       <property name="placeholderText">
        <string>paste ID of the inviter</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_5">
     <item>
      <widget class="QLabel" name="email_label">
       <property name="text">
        <string>Email:</string>
       </property>
       <property name="buddy">
        <cstring>secret_line</cstring>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="email_line">
       <property name="placeholderText">
        <string>example@email.com</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_4">
     <item>
      <widget class="QLabel" name="group_label">
       <property name="text">
        <string>Group:</string>
       </property>
       <property name="buddy">
        <cstring>secret_line</cstring>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="group_line">
       <property name="placeholderText">
        <string>group server URL...</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_6">
     <item>
      <widget class="QLabel" name="group_key_label">
       <property name="text">
        <string>Group Key:</string>
       </property>
       <property name="buddy">
        <cstring>secret_line</cstring>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="group_key_line">
       <property name="placeholderText">
        <string>group server public key...</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="secret_label">
       <property name="text">
        <string>Secret:</string>
       </property>
       <property name="buddy">
        <cstring>secret_line</cstring>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="secret_line">
       <property name="placeholderText">
        <string>paste secret code...</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QLabel" name="password_label">
       <property name="text">
        <string>Password:</string>
       </property>
       <property name="buddy">
        <cstring>password_line</cstring>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="password_line">
       <property name="maxLength">
        <number>256</number>
       </property>
       <property name="echoMode">
        <enum>QLineEdit::Password</enum>
       </property>
       <property name="placeholderText">
        <string>type secure password...</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>RegisterDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>184</x>
     <y>278</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>RegisterDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>184</x>
     <y>284</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
